doc/user/interop.md

```diff
@@ -48,14 +48,14 @@ Call `size` on the object.
 
 ### `IS_BOXED`
 
-Returns true only for instances of `String`, `Symbol`, `Rubinius::FFI::Pointer`
-and objects that respond to `unbox`.
+Returns true only for instances of `String`, `Symbol`,
+`Truffle::Support::FFI::Pointer` and objects that respond to `unbox`.
```

Comment:
Can we simplify Truffle::Support::FFI to just Truffle::FFI? It's way too long.

Reply:
Yes, as I've said we can move things into `Truffle`, I just haven't done it yet.